tools/malwarejail/env/wscript.js

```diff
@@ -1305,7 +1309,10 @@ WScript_Shell = function () {
         REG[_reg_normalize(a)] = b;
     }
     this.regread = function (a) {
-        var ret = "" + REG[_reg_normalize(a)];
+        var normalized_registry_key = _reg_normalize(a);
+        // Remove trailing backwards slashes if they exist
+        normalized_registry_key = normalized_registry_key.replace(/\\+$/, "");
```

Comment:
Wouldn't this make the key on line 1270 inaccessible? (since it ends with a trailing backslash)

Reply:
good catch, thanks!